app/src/components/EditEvent/EditEvent/EditEvent.tsx

```diff
@@ -341,49 +341,63 @@ export const EditEvent = ({ eventResult: event, updateEvent }: IProps) => {
           )}
         </div>
 
-        {event.participantQuestion !== undefined ? (
-          <div>
+        <div>
+          {event.participantQuestions.map((q, i) => (
             <ValidatedTextArea
               label={labels.participantQuestion}
               placeholder={placeholders.participantQuestion}
-              value={event.participantQuestion}
-              validation={parseQuestion}
+              value={q}
+              validation={(s) => parseQuestions([s])}
               onLightBackground
               minRow={4}
               onChange={(participantQuestion) =>
                 updateEvent({
                   ...event,
-                  participantQuestion,
+                  participantQuestions: event.participantQuestions.map(
+                    (oldQ, oldI) => {
+                      if (i === oldI) {
+                        return participantQuestion;
+                      }
+                      return oldQ;
+                    }
+                  ),
                 })
               }
             />
+          ))}
+          {event.participantQuestions.length > 0 && (
             <Button
               className={style.removeQButton}
               displayAsLink
               onLightBackground
               onClick={() =>
-                updateEvent({ ...event, participantQuestion: undefined })
+                updateEvent({
+                  ...event,
+                  participantQuestions: event.participantQuestions
+                    .reverse()
+                    .slice(1)
+                    .reverse(),
```

Comment:
Is it worth it? Let me work it
I put my thang down, flip it and reverse it
Ti esrever dna ti pilf nwod gnaht ym tup i
Ti esrever dna ti pilf nwod gnaht ym tup i

Reply:
Hahah whaat iis this?

Reply:
https://www.youtube.com/watch?v=cjIvu7e6Wq8